api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -42,6 +42,7 @@
 
 import static gov.cms.ab2d.api.util.Constants.GENERIC_FHIR_ERR_MSG;
 import static gov.cms.ab2d.api.util.SwaggerConstants.*;
+import static gov.cms.ab2d.common.service.JobServiceImpl.ZIPFORMAT;
```

Comment:
Should we move this constant to the `JobService` interface? That way you could avoid referencing an implementation class directly.